src/main/java/org/folio/service/upload/FileUploadService.java

```diff
@@ -40,8 +40,8 @@
   /**
    * Completes uploading for the given {@link FileDefinition}
    *
-   * @param fileDefinition  {@link FileDefinition}
-   * @param tenantId tenant id
+   * @param fileDefinition {@link FileDefinition}
```

Comment:
Not part of this PR, but why is the FileUploadService creating a FileDefinition?
Shouldn't it be part of FileDefinitionService?